python/tank/path_cache.py

```diff
@@ -1265,3 +1170,142 @@ def get_secondary_entities(self, path):
 
         return matches
     
+
+    def ensure_all_entries_are_in_shotgun(self, log):
+        """
+        Ensures that all the path cache data in this database is also registered in Shotgun.
+        
+        This will go through each entity in the path cache database and check if it exists in 
+        Shotgun. If not, it will be created.
+        
+        No updates will be made to the path cache database.
+        
+        :param log: Std python logger 
+        """
+
+        SG_BATCH_SIZE = 16
```

Comment:
This batch size is much smaller than the previous batch size (400) - is there a reason for this or am I misreading the previous code?